homeassistant/components/media_player/kef.py

```diff
@@ -0,0 +1,302 @@
+"""
+Support for interfacing HA with the KEF Wireless Speakers .
+
+For more details about this platform, please refer to the documentation at
+https://github.com/Gronis/pykef
+"""
+
+
+import collections
+import logging
+import time
+import json
+from enum import Enum
+import voluptuous as vol
```

Comment:
Import must be grouped.

Reply:
Not sure what you mean. Grouping from and import from each other like this?
```suggestion
from enum import Enum
import time
import json
import voluptuous as vol
```